ext/pcre/php_pcre.c

```diff
@@ -2479,7 +2479,11 @@ PHP_FUNCTION(preg_replace_callback_array)
 	}
 
 	if (subject_ht) {
-		RETURN_ARR(subject_ht);
+		RETVAL_ARR(subject_ht);
+		if (GC_FLAGS(subject_ht) & GC_IMMUTABLE) {
```

Comment:
```suggestion
		if (GC_FLAGS(subject_ht) & IS_ARRAY_IMMUTABLE) {
```
Maybe?

Reply:
Right, I forgot we have an alias. Thanks!